### ISSUE TITLE
Enable notifications - design and wording change?

### ISSUE THREAD

**Issue by Cleop** [emfoundation/ce100-app#1272]
![image](https://user-images.githubusercontent.com/16775804/35913886-22e5bb02-0bf9-11e8-9316-40b6872f9d77.png)

On the 'Edit use details' page in #899 a consent for marketing and newsletters toggle is being added and the page now looks like the image above. Does the wording/ design or the email notifications checkbox need to change so that it's clear exactly what is meant by enable notifications?

**Comment by asimonw** [comment]
I think that something a bit more wordy might be useful, but maybe we could add some text above the checkbox instead. 'Challenge updates' is a bit generic because they'll only get notified when someone comments. Maybe "Would you like to be notified when someone comments on one of your challenges?" That might be a bit too wordy. @fannyBkane any thoughts on what would work well with the brand voice?

**Comment by fannyBkane** [comment]
Hey @asimonw what about if we change 'Enable notifications' to 'I'd like to be notified when someone comments on one of my challenges'?

**Comment by asimonw** [comment]
@Cleop I'm happy with @fannyBkane 's suggestion.
